traitquestdev/companylogin.php

```diff
@@ -21,19 +33,32 @@
 </head>
 
 <body>
-	<h1>Admin Login</h1>
-	<form id="formCompanyLogin" class="form" method="post">
-		<div id="columnCompany">
-			<input type="text" name="company" id="companyName" class="inputForm" placeholder="Company Name" />
-		</div>
-		<div id="columnEmail">
-			<input type="text" name="email" id="email" class="inputForm" placeholder="Email" />
-		</div>
-		<div id="columnPassword">
-			<input type="password" name="password" id="password" class="inputForm" placeholder="Password" />
-		</div>
-		<div id="loginResponse"></div>
-		<input type="submit" name="submit" id="loginSubmit" class="buttonForm" value="Login" />
-	</form>
+<div id="wrapper">
+	<?php include "header.php"; ?>
+	<div id="mainContainer" class="container">
+		<h1 class="text-center" hidden>Admin Login</h1>
+		<form id="formCompanyLogin" class="form padding-topbottom-s" method="post">
+			<div id="columnCompany" class="columnInput">
+				<div class="icon-addon">
+					<input type="text" name="company" id="companyName" class="inputForm padding-left30px" placeholder="Company" />
+					<i class="glyphicon glyphicon-briefcase"></i>
+				</div>
+			</div>
+			<div id="columnEmail" class="columnInput">
+				<div class="icon-addon">
+					<input type="text" name="email" id="email" class="inputForm padding-left30px" placeholder="Email" />
+					<i class="glyphicon glyphicon-envelope"></i>
+				</div>
+			</div>
+			<div id="columnPassword" class="columnInput">
+				<div class="icon-addon">
+					<input type="password" name="password" id="password" class="inputForm padding-left30px" placeholder="Password" />
+					<i class="glyphicon glyphicon-lock"></i>
+				</div>
+			</div>
+			<div id="loginResponse"></div>
+			<input type="submit" name="submit" id="loginSubmit" class="buttonForm button" value="Login As Admin" />
+		</form>
+	</div>
```

Comment:
to include footer.php but remove address, social network icons etc from footer.php